src/dialects/abstract/query-interface.js

```diff
@@ -975,8 +975,13 @@ class QueryInterface {
 
   async select(model, tableName, optionsArg) {
     const options = { ...optionsArg, type: QueryTypes.SELECT, model };
+    const query = this.QueryGenerator.selectQuery(tableName, options, model);
```

Comment:
looks like this should be `const query = this.queryGenerator.selectQuery(tableName, options, model);`